app/Resources/views/admin/blog/show.html.twig

```diff
@@ -46,5 +46,7 @@
         }, with_context = false) }}
     </div>
 
+    {{ parent() }}
```

Comment:
I don't remember ... but the rest of admin/* templates include this `parent()` ... so this is probably a bug.